recipe/meta.yaml

```diff
@@ -35,13 +33,14 @@ requirements:
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - shapely >=1.5.6
-    - pyshp >=1.1.4
+    - shapely >=1.6.4
+    - pyshp >=2.1
     - six >=1.3.0
-    - proj
-    - geos
+    - proj >=8.0
```

Comment:
@akrherz, thank you for getting this going!  `proj` should not have a constraint because we use conda-forge's pins.  The same it true of `geos`.  `pyproj` is fine.

Reply:
```suggestion
    - proj
```

Reply:
Done and thanks, my wild thrashing aside :(